src/app/components/media-viewer-toolbar/media-viewer-toolbar.component.scss

```diff
@@ -0,0 +1,242 @@
+
+.jui-toolbar {
+  background-color: #ffffff;
+  border-bottom: 1px solid #DDDDDD;
+  border-top: 1px solid #DDDDDD;
+  position: sticky;
+  position: -webkit-sticky;
+  top: -1px;
+  width: 100%;
+  z-index: 200;
+  height: 40px;
+}
+
+.jui-toolbar::before {
+  content: "";
+  background-color: #ffffff;
+  height: 100%;
+  position: absolute;
+  width: 100%;
+  z-index: 0;
+}
+
+.jui-toolbar::after {
+  content: "";
+  display: block;
+  height: 1px;
+  left: 50%;
+  margin-top: 0;
+  margin-left: -50%;
+  position: absolute;
+  width: 100%;
+  z-index: -1;
+  -webkit-box-shadow: 0px 0px 3px 2px #BEBEBE;
+  -moz-box-shadow: 0px 0px 3px 2px #BEBEBE;
+  box-shadow: 0px 0px 3px 2px #BEBEBE;
+}
+
+.mv-toolbar__container {
+  border-left: 1px solid #dee0e2;
+  border-right: 1px solid #dee0e2;
+  height: 100%;
+  position: relative;
+  left: 0;
+  top: 0;
+  z-index: 1;
+}
+
+.mv-toolbar-left {
+  float: left;
+}
+
+.mv-toolbar-right {
+  float: right;
+}
+
+.mv-toolbar-center {
+  position: absolute;
+  left: 50%;
+  -webkit-transform: translateX(-50%);
+  transform: translateX(-50%);
+}
+.mv-toolbar__container::after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.jui-toolbar__button {
+  font-family: "nta", Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
I appreciate the  fact that this is copied code, but why are the values different per vendor here? Also is there no base `font-smoothing` property?